packages/core/src/components/Alert/styles.ts

```diff
@@ -0,0 +1,65 @@
+import { StyleSheet } from '../../hooks/useStyles';
+
+const styleSheet: StyleSheet = ({ color, pattern, unit }) => ({
```

Comment:
Maybe we use named exports everywhere? Just for situations where there may be multiple sheets.

Reply:
i was thinking `styleSheet` for `index` styles. and then for multiple sheets, used the file name as a suffix, e.g. for Item in Accordion: `styleSheetItem`.  thoughts on that?

Reply:
Yeah sounds. Just think we should align all styles to use named exports.

Reply:
@milesj cool - ill work on doing this approach for the others

Reply:
done!